src/main/java/org/learn2code/seoul/pomodoro/user/repository/UserRepositoryJdbc.java

```diff
@@ -0,0 +1,224 @@
+package org.learn2code.seoul.pomodoro.user.repository;
+
+import org.learn2code.seoul.pomodoro.user.domain.User;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserRepositoryJdbc implements UserRepository {
+
+	private static final String URL = "jdbc:h2:tcp://localhost/~/test";
+
+	@Override
+	public void create(User user) {
+		int rows = 0;
+		Connection conn = null;
+		Statement stmt = null;
+		ResultSet rs = null;
+
+		try {
+			conn = DriverManager.getConnection(URL);
+			stmt = conn.createStatement();
+			String email = user.getEmail();
+			String password = user.getPassword();
+			String sql = "INSERT INTO users VALUES (null, '" + email + "', '" + password +"')";
+			System.out.println(sql);
+			rows = stmt.executeUpdate(sql);
```

Comment:
okay neat, so that statement executes the update, then we can use rs of type resultset to get the generated keys (in our case the id field / primary key), and set that user's ID on the programs side.